dotcom-rendering/src/web/components/SecureSignupIframe.importable.tsx

```diff
@@ -100,49 +101,54 @@ const postFormData = async (
 	});
 };
 
+type EventDescription =
+	| 'click-button'
+	| 'form-submission'
+	| 'submission-confirmed'
+	| 'submission-failed'
+	| 'open-captcha'
+	| 'captcha-load-error'
+	| 'form-submit-error'
+	| 'captcha-not-passed'
+	| 'captcha-passed';
+
 const sendTracking = (
 	newsletterId: string,
-	event:
-		| 'click-button'
-		| 'submit-form'
-		| 'form-submit-error'
-		| 'captcha-load-error'
-		| 'open-captcha'
-		| 'captcha-not-passed'
-		| 'captcha-passed'
-		| 'success-response-received'
-		| 'fail-response-received',
+	eventDescription: EventDescription,
 ): void => {
 	const ophanRecord = getOphanRecordFunction();
 
 	let action: OphanAction = 'CLICK';
```

Comment:
Slightly out of scope for this PR, but if you want to avoid mutations, you could:

```ts
const getAction = (event: EventDescription): OphanAction => {
	switch(event) {
		case 'form-submission':
		case 'form-submission':
			return 'SUBSCRIBE';
		/* … */
		default:
			return 'CLICK';
 } 
}

/* later */
const action = getAction(eventDescription);
```